ui/candidate-portal/src/app/model/form.ts

```diff
@@ -44,3 +45,39 @@ export interface TravelDocFormData {
   travelDocIssueDate: string;
   travelDocExpiryDate: string;
 }
+
+export interface FamilyMemberDoc {
```

Comment:
Won't this be the same as the same structure for the primary applicant? Do we need a special interface for it?